go.mod

```diff
@@ -157,3 +157,5 @@ require (
 	k8s.io/kms v0.28.4 // indirect
 	k8s.io/kube-aggregator v0.28.2 // indirect
 )
+
+replace github.com/openshift/api => ../api/
```

Comment:
this is actually
https://github.com/yuvalk/api/tree/cherry-pick-1667-to-release-4.15